core/perf_test/PerfTest_ViewCopy.hpp

```diff
@@ -42,220 +42,154 @@
 //@HEADER
 */
 
+#ifndef KOKKOS_CORE_PERFTEST_BENCHMARK_VIEW_COPY_HPP
+#define KOKKOS_CORE_PERFTEST_BENCHMARK_VIEW_COPY_HPP
+
 #include <Kokkos_Core.hpp>
-#include <gtest/gtest.h>
-#include <cstdio>
-#include <PerfTest_Category.hpp>
+#include <benchmark/benchmark.h>
+#include <cmath>
 
 namespace Test {
 
+/**
+ * \brief Mark the label as a figure of merit.
+ */
+inline std::string benchmark_fom(const std::string& label) {
+  return "FOM: " + label;
+}
+
+inline void report_results(benchmark::State& state, std::size_t num_elems,
+                           double time) {
+  state.SetIterationTime(time);
+
+  // data size in megabytes
+  const auto size = 1.0 * num_elems * sizeof(double) / 1024 / 1024;
+  // data processed in gigabytes
+  const auto data_processed = 2 * size / 1024;
+
+  state.counters["MB"] = benchmark::Counter(size, benchmark::Counter::kDefaults,
+                                            benchmark::Counter::OneK::kIs1024);
+  state.counters[benchmark_fom("GB/s")] = benchmark::Counter(
```

Comment:
Single use of `benchmark_fom` that yields a convoluted way or spelling out `"FOM: GB/s"`

Reply:
This comes from the "Figure of Merit" conversation that we've had. Note that there are more converted tests / benchmarks incoming in subsequent reviews, so this would become a common helper.
On the other hand, I don't see anything wrong with spelling "FOM: GB/s" directly, so I could remove it as well.